### PR TITLE
added logic for calculating timezone offset, changes for moengage and…

### PR DESCRIPTION
**Pull request by ruchiramoitra**
… updated test cases

## Description of the change

> Description here

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

**Comment by sayan-mitra** [comment]
looks good 👍

**Review by sayan-mitra — APPROVED** [review]

**Review by prabrishac — APPROVED** [review]
👍

File: package.json [hunk 1]
```diff
@@ -47,12 +47,13 @@
...
     "minimist": ">=0.2.1",
     "moment": "^2.27.0",
+    "moment-timezone": "^0.5.32",
     "node-cache": "^5.1.2",
     "node-fetch": "^2.6.1",
-    "ua-parser-js": "^0.7.22",
     "set-value": "^3.0.2",
     "sha256": "^0.2.0",
     "statsd-client": "^0.4.4",
+    "ua-parser-js": "^0.7.22",
```
Comment (arnab-p) [inline package.json:56]: Don't we have it already in place?

**Review by arnab-p — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (ruchiramoitra) [reply package.json:56]: that got deleted and added it again... same thing...

Comment (arnab-p) [reply package.json:56]: Sorry! my bad. 👍

**Review by arnab-p — APPROVED** [review]

**Pull request merged**